src/services/clinical/careplan.service.ts

```diff
@@ -575,6 +584,65 @@ export class CareplanService implements IUserActionService {
         return await this._careplanRepo.updateActivityUserResponse(activityId, userResponse);
     };
 
+    public updateRisk = async (updateRisk: EnrollmentDomainModel): Promise<EnrollmentDto> => {
+
+        const filter = {
+            Phone : updateRisk.Phone
+        };
+        const patient = await this._patientRepo.search(filter);
+        if (patient.Items.length === 0) {
+            throw new Error('Patient does not exist!');
+        }
+        updateRisk.PatientUserId = patient.Items[0].UserId;
+
+        return  await this._careplanRepo.updateRisk(updateRisk);
+    };
+
+    public scheduleDailyHighRiskCareplan = async (): Promise<void> => {
```

Comment:
This function should not be part of the careplan.service.ts. 
It should be moved to the plugins / custom handlers.

Reply:
Done